packages/front-end/components/FactTables/FactMetricModal.tsx

```diff
@@ -235,7 +330,7 @@ function ColumnRefSelector({
                 label: f.name,
                 value: f.id,
               }))}
-              placeholder="All Rows"
+              placeholder={inlineFilterFields.length > 0 ? "None" : "All Rows"}
```

Comment:
WDYT a "test" functionality on the inline filter fields to make sure you can get some data, especially if you're creating a field (since this is now harder)? Maybe it's off by default, or it's literally a "test" button that shows up when you Create a new field?

IDK may be overkill, but it could be easy to misspecify this field and I use the test function for Filters all the time.

Reply:
Yeah, a test button when creating a metric makes sense, even before this PR. Even if we just show a count of matched events in the past X days.

Will probably do this in a separate PR